packages/web/integrations/axelar/types.ts

```diff
@@ -37,6 +38,7 @@ export const EthClientChainIds_AxelarChainIdsMap: {
   "Avalanche C-Chain": "Avalanche",
   "Binance Smart Chain": "Binance",
   "Fantom Opera": "Fantom",
+  "Moonbase": "Moonbeam",
```

Comment:
It's called moonbase on Metamask?

Reply:
Moonbeam's docs call it 'Moonbase Alpha':
https://docs.moonbeam.network/builders/get-started/networks/moonbase/
Axelar docs call it 'Moonbase':
https://docs.axelar.dev/resources/testnet

Which is the preferred source of truth?

Reply:
https://github.com/osmosis-labs/osmosis-frontend/blob/JP69/stage-erc-axl-xfer/packages/web/integrations/ethereum/types.ts#L28

Actually, it doesn't matter to me, we just need to make sure we add `"Moonbase"` as an entry in this mapping of Ethereum chain IDs (given from MetaMask/EVM wallets) to display names. These mappings are actually related in the code. This isn't clear though; could you please add to the comment above `EthClientChainIds_AxelarChainIdsMap` to clarify this? That the entries are related?

https://github.com/osmosis-labs/osmosis-frontend/blob/0aadd61fc774fc244dd382b11406d45fd2572e68/packages/web/integrations/ethereum/types.ts#L28-L44

Reply:
Added comments. and updated Testnet name for Moonbase -> Moonbase Alpha. and removed ropsten -> Ethereum mapping